<?xml version="1.0" encoding="UTF-8"?>
<testsuite errors="0" failures="2" name="notify.tests.TestApi-20171219105741" skipped="0" tests="10" time="1.163">
	<testcase classname="notify.tests.TestApi" name="test_BaseURL" time="0.062"/>
	<testcase classname="notify.tests.TestApi" name="test_PostBadEmailReq" time="0.000"/>
	<testcase classname="notify.tests.TestApi" name="test_PostBadSMSReq" time="0.000"/>
	<testcase classname="notify.tests.TestApi" name="test_PostMissingEmailReq" time="0.016"/>
	<testcase classname="notify.tests.TestApi" name="test_PostMissingSMSReq" time="0.000"/>
	<testcase classname="notify.tests.TestApi" name="test_SwaggerURL" time="0.001"/>
	<testcase classname="notify.tests.TestApi" name="test_badSetApiKey" time="0.000"/>
	<testcase classname="notify.tests.TestApi" name="test_setApiKey" time="0.017"/>
	<testcase classname="notify.tests.TestApi" name="test_PostEmailReq" time="0.834">
		<failure message="429 != 201" type="AssertionError">
<![CDATA[Traceback (most recent call last):
  File "C:\Users\stylesm\Desktop\test\OFS-MORE-Beta-Secured\source\notify\notify\tests.py", line 82, in test_PostEmailReq
    self.assertEqual(response.status_code, 201)
AssertionError: 429 != 201

Stderr:
API POST request on https://api.notifications.service.gov.uk/v2/notifications/email failed with 429 '[{'message': 'Exceeded send limits (50) for today', 'error': 'TooManyRequestsError'}]'
]]>		</failure>
	</testcase>
	<testcase classname="notify.tests.TestApi" name="test_PostSMSReq" time="0.234">
		<failure message="429 != 201" type="AssertionError">
<![CDATA[Traceback (most recent call last):
  File "C:\Users\stylesm\Desktop\test\OFS-MORE-Beta-Secured\source\notify\notify\tests.py", line 97, in test_PostSMSReq
    self.assertEqual(response.status_code, 201)
AssertionError: 429 != 201

Stderr:
API POST request on https://api.notifications.service.gov.uk/v2/notifications/sms failed with 429 '[{'message': 'Exceeded send limits (50) for today', 'error': 'TooManyRequestsError'}]'
]]>		</failure>
	</testcase>
	<system-out>
<![CDATA[]]>	</system-out>
	<system-err>
<![CDATA[API POST request on https://api.notifications.service.gov.uk/v2/notifications/email failed with 429 '[{'message': 'Exceeded send limits (50) for today', 'error': 'TooManyRequestsError'}]'
API POST request on https://api.notifications.service.gov.uk/v2/notifications/sms failed with 429 '[{'message': 'Exceeded send limits (50) for today', 'error': 'TooManyRequestsError'}]'
]]>	</system-err>
</testsuite>
